### DISCUSSION TITLE
Can't install WebView2

### DISCUSSION THREAD

**Discussion by babakfp** [tauri-apps/tauri#10133] [Q&A]
Trying to install WebView2 downloaded from Microsoft:

![image](https://github.com/tauri-apps/tauri/assets/44144724/906b7f84-082e-46f4-803e-d4cb5ce9890a)

![image](https://github.com/tauri-apps/tauri/assets/44144724/42d357a6-0a34-4296-8651-e110aeb1cda8)

---

Trying to install Edge browser:

![image](https://github.com/tauri-apps/tauri/assets/44144724/8ae83073-9197-4989-8595-f5fcd0318875)

---

There was a time I couldn't get this crap out of my system, now I can't get it into the system 😂.

There was a folder in `C:\Program Files/Microsoft/Edge` and `C:\Program Files (x86)/Microsoft/Edge` and I deleted the whole thing, because f Edge. This is why I'm having issues with webview now!

**Comment by SpikeHD** [comment]
I've had a bunch of people with this issue in a couple different projects, I believe when the installer says WebView2 is already installed (when it actually isn't) you can delete this registry key and try again:

`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}`

If that doesn't work, you can also try [uninstalling it via CMD](https://superuser.com/questions/1743585/how-to-uninstall-microsoft-edge-webview2/1743626#1743626).
